etc/fish_completion/kw.fish

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env fish
```

Comment:
Does not make sense to add this file at `/etc/fish_completition/kw.fish`. Why did you add it in the `src` directory? I recommend you to follow the same patter used by bash.

Reply:
This answers one of my questions :-). I didn't put it in `src` because it's not part of kw itself and I thought it could mess that repository. About using the same pattern used by bash (maybe `fish_autocomplete.fish` ?), I discussed that in the previous conversation.